packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,100 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: -2px;
+  left: calc(var(--hds-app-sidenav-border-width) * -1);
   z-index: 1;
+  display: flex;
+  box-sizing: content-box;
+  padding: calc(var(--token-side-nav-toggle-button-padding-vertical) - var(--hds-app-sidenav-border-width)) var(--token-side-nav-toggle-button-padding-horizontal);
   color: var(--token-color-foreground-high-contrast);
+  text-align: right;
   background: none;
-  border: 1px solid transparent;
+  background-color: var(--token-side-nav-color-surface-primary);
+  border: var(--hds-app-sidenav-border-width) solid var(--hds-app-sidenav-border-color);
+  border-left-color: transparent;
+  border-top-right-radius: var(--hds-app-sidenav-border-radius);
+  border-bottom-right-radius: var(--hds-app-sidenav-border-radius);
+  transform:
+    translateX(var(--hds-app-sidenav-width-expanded))
+    translateY(24px);
   cursor: pointer;
-  transition-timing-function: var(--hds-app-sidenav-animation-easing);
-  transition-duration: var(--hds-app-sidenav-animation-duration);
-  transition-property: width, height, padding, border-radius, transform;
+  transition:
+    transform var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing),
+    padding var(--hds-app-sidenav-animation-duration-toggle-button)
+    var(--hds-app-sidenav-animation-easing);
+
+  &::before {
+    position: absolute;
+    top: calc(var(--hds-app-sidenav-border-radius) * -2);
+    left: calc(var(--hds-app-sidenav-border-width) * -1);
+    box-sizing: border-box;
+    width: calc(var(--hds-app-sidenav-border-radius) * 2);
+    height: calc(var(--hds-app-sidenav-border-radius) * 2);
+    color: var(--token-color-foreground-high-contrast);
```

Comment:
are you sure this is needed? 🤔

Reply:
leftover, removed it 531c5b78cad742c6695961175c3f5afcd62334e3